packages/usa-select/src/usa-select.twig

```diff
@@ -7,3 +7,4 @@
     <option value="value3">Option C</option>
   </select>
 </form>
+
```

Comment:
This one also shows an extra line at end.

![image](https://user-images.githubusercontent.com/3385219/173589668-6f0c6626-b1d6-457d-9622-964d23f935ad.png)